src/components/About.css

```diff
@@ -0,0 +1,112 @@
+.about-container {
+    background-image: url(../images/backgrounds/About_Bg.png);
```

Comment:
Set more background css settings for how picture loads onto screen. Each browsers default settings are different.